<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inbox</class>
 <widget class="QWidget" name="inbox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QScrollArea" name="scrollArea_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>651</width>
     <height>541</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>649</width>
      <height>539</height>
     </rect>
    </property>
    <widget class="QScrollBar" name="verticalScrollBar">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>0</y>
       <width>20</width>
       <height>541</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QScrollArea" name="scrollArea_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>631</width>
       <height>23</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>629</width>
        <height>21</height>
       </rect>
      </property>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>383</x>
         <y>0</y>
         <width>81</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Move to</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>inbox</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>send</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>new folder</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="pushButton_7">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>80</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_8">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>0</y>
         <width>80</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Forward</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_9">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>0</y>
         <width>141</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Mark all as read</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_2">
       <property name="geometry">
        <rect>
         <x>302</x>
         <y>0</y>
         <width>81</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Mard as</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>read</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>unread</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>star</string>
        </property>
       </item>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>612</x>
     <y>20</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Search Messages</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>21</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Inbox</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QTreeWidget" name="treeWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>151</width>
     <height>551</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <column>
    <property name="text">
     <string>Func</string>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <item>
    <property name="text">
     <string>New</string>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </item>
   <item>
    <property name="text">
     <string>All</string>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Read</string>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Unread</string>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Flagged</string>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Deleted</string>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Inbox Floder</string>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <item>
     <property name="text">
      <string>新建项目</string>
     </property>
     <item>
      <property name="text">
       <string>新建子项目</string>
      </property>
     </item>
    </item>
    <item>
     <property name="text">
      <string>新建项目</string>
     </property>
     <item>
      <property name="text">
       <string>新建子项目</string>
      </property>
     </item>
    </item>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_quit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>20</y>
     <width>56</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>quit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_logout">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>20</y>
     <width>56</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>logout</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_write">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>56</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>write</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
